### PR TITLE
README: Use SVG for build badges

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR changes the build badges to SVG for sharper readability ✨

### CODE REVIEW

**Comment by jkeam** [comment]
build errors :(

**Comment by olleolleolle** [comment]
...but it errors with clarity and sharpness in the badge!

**Comment by jkeam** [comment]
lol haha

**Comment by olleolleolle** [comment]
Cc @bronzdoc 👋

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
+# MetricFu [![Gem Version](https://badge.fury.io/rb/metric_fu.svg)](http://badge.fury.io/rb/metric_fu) [![Build Status](https://travis-ci.org/metricfu/metric_fu.svg?branch=master)](http://travis-ci.org/metricfu/metric_fu)
 
 [![Join the chat at https://gitter.im/metricfu/metric_fu](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/metricfu/metric_fu?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-[![Code Climate](https://codeclimate.com/github/metricfu/metric_fu.png)](https://codeclimate.com/github/metricfu/metric_fu) [![Inline docs](http://inch-ci.org/github/metricfu/metric_fu.png)](http://inch-ci.org/github/metricfu/metric_fu) [![Dependency Status](https://gemnasium.com/metricfu/metric_fu.png)](https://gemnasium.com/metricfu/metric_fu)
-
+[![Code Climate](https://codeclimate.com/github/metricfu/metric_fu.svg)](https://codeclimate.com/github/metricfu/metric_fu) [![Inline docs](http://inch-ci.org/github/metricfu/metric_fu.svg)](http://inch-ci.org/github/metricfu/metric_fu)
 [Rdoc](http://rdoc.info/github/metricfu/metric_fu/)
```
Comment (bronzdoc) [inline README.md:6]: I think you need a space here so `Rdoc` shows in the next line

**Comment by bronzdoc** [comment]
@olleolleolle 👋 

Just one small comment :)

**Comment by olleolleolle** [comment]
@bronzdoc Thanks for taking a look!

**Pull request merged**